examples/01_tutorials/plot_single_subject_single_run.py

```diff
@@ -205,12 +205,12 @@
 # statistical validity of the procedure: here we used an arbitrary
 # threshold of 3.0 but the threshold should provide some guarantees on
 # the risk of false detections (aka type-1 errors in statistics). One
-# first suggestion is to control the false positive rate (fpr) at a
-# certain level, e.g. 0.001: this means that there is.1% chance of
-# declaring active an inactive voxel.
+# first suggestion is to control the false positive rate (fpr, denoted
```

Comment:
One first suggestion? That is a weird statement. Maybe we mean One suggestion here?  
denoted by alpha will be better than denoted alpha